apis/betterself/v1/utils/views.py

```diff
@@ -26,3 +26,15 @@ def get_queryset(self):
 class BaseGenericListCreateAPIViewV1(ListCreateAPIView, UserQuerysetFilterMixin):
     def get_queryset(self):
         return self._get_queryset()
+
+
+class ReadOrWriteViewInterfaceV1(object):
```

Comment:
See if there are other mixin/interfaces that you've done with this pattern that you might be able to merge.